src/tsickle.ts

```diff
@@ -1222,6 +1222,11 @@ class Annotator extends ClosureRewriter {
     const forwardDeclarePrefix = `tsickle_forward_declare_${++this.forwardDeclareCounter}`;
     const moduleNamespace =
         nsImport !== null ? nsImport : this.host.pathToModuleName(this.file.fileName, importPath);
+    const moduleSymbol = this.typeChecker.getSymbolAtLocation(specifier);
+    // Modules that do not have exports do not have a symbol. These can still be imported, either as
```

Comment:
Adjusted the comment. Indeed this only happens for scripts - if the imported is a module, it'll have a symbol. What happens is that importing an empty set of symbols from a script causes its global declarations to be around, but does not emit a load statement, which is what people appear to use it for.

I'm not even sure if that's a bug in TS or intended behaviour, but we should definitely not crash on it, and it currently works in TS.